lib/graphql/account.ts

```diff
@@ -28,4 +49,14 @@ export class Account extends Model<AccountModel> {
     const result: Query = await this.client.rawQuery(GET_ACCOUNT);
     return result.viewer?.mainAccount || null;
   }
+
+  /**
+   * Different statistics to the users account balance e.g. yours, unknow, vatTotal, taxMissing ...
+   *
+   * @returns current user account statistics
+   */
+  public async getAccountStats(): Promise<AccountStats | null> {
```

Comment:
Also I was thinking of making it configurable by passing an array of strings with fields you really want to have,
e.g `getAccountStats(['accountBalance','vatTotal','taxTotal'])`. And if no parameter is provided, return all fields like it is at the moment. But not sure if we should keep it simple. What do you think about it  🤔

Reply:
This is an idea, but at that point, consumers would be probably better off writing the query on their own with rawQuery directly.

For now we don't provide such possibility of filtering in any other helper. I'd say we should keep it that way for now. 

We will always have the possibility later on to enrich these helpers, e.g. `getStats({ fields: [...] })` or so.